client/src/components/BillSummaryPage.jsx

```diff
@@ -60,8 +59,6 @@ class BillSummaryPage extends React.Component {
           Tip Percentage (%):
           <input type="number" id="tipPercentage" name="tipPercentage" min="0" max="1000" value={this.props.tipPercentage} onChange={this.changeTipPercentage}/><br/>
         </form>
-        <button onClick={this.splitEvenly}>Split Evenly</button>
-        <button onClick={this.splitByItem}>Split by Item</button>
```

Comment:
Why are we removing these buttons? Don't we still need them?

Reply:
They are doubled. See further down for the buttons.